lumicks/pylake/detail/imaging_mixins.py

```diff
@@ -72,3 +77,30 @@ def plot(num):
         )
         line_ani.save(file_name, writer=writer)
         plt.close(fig)
+
+
+class FrameIndex:
+    def to_frame_index(self, time, use_start=False) -> Union[int, None]:
+        """Convert timestamps, timestamps or frame indices to frame indices
+
+        Parameters
+        ----------
+        time : str or int or None
+            Time expressed in either string format (e.g. "5s"), timestamp in nanoseconds or frame.
+        use_start : bool
+            Are we dealing with the start or end of a frame here?
```

Comment:
The term "use" here seems odd, I wasn't quite sure what the parameter meant. I think "is" might be better and more in line with convention for naming boolean args.

The description seems a bit off also; until I looked at the usage in the code I was thinking this arg meant something completely different. Not sure if I quite got the intended usage, but how about:
```suggestion
        is_frame_start : bool
            Whether the input time refers to the start or stop timestamp of the searched frame
```

Reply:
I agree with your suggestion. I think your names and docstring description are better here.